test/SmokeTests/e2e-test.ps1

```diff
@@ -0,0 +1,94 @@
+﻿# Installing PowerShell: https://docs.microsoft.com/powershell/scripting/install/installing-powershell
+
+param(
+	[Parameter(Mandatory=$true)]
+	[string]$DockerfilePath,
+	[Parameter(Mandatory=$true)]
+	[string]$HttpStartPath,
+    [string]$ImageName="dfapp",
+	[string]$ContainerName="app",
+	[switch]$NoSetup=$false,
+	[switch]$NoValidation=$false,
+	[int]$Sleep=30
+)
+
+$ErrorActionPreference = "Stop"
+
+if ($NoSetup -eq $false) {
+	# Build the docker image first, since that's the most critical step
+	Write-Host "Building sample app Docker container from '$DockerfilePath'..." -ForegroundColor Yellow
+	docker build -f $DockerfilePath -t $ImageName $PSScriptRoot/../../
+
+	# Next, download and start the Azurite emulator Docker image
+	Write-Host "Pulling down the mcr.microsoft.com/azure-storage/azurite image..." -ForegroundColor Yellow
+	docker pull mcr.microsoft.com/azure-storage/azurite
```

Comment:
Is there a reason why we don't build a Docker image with Azurite pre-installed in it? It might save time during automation...but I wouldn't block this PR based on that. Just an idea

Reply:
That would need to be a Docker image that's preconfigured for Azure Functions development scenarios. We wouldn't want this for production Docker images since that would bloat the image size and hurt cold start performance. This is probably a good question for the team that's working on the Azure Functions tooling experience, though (assuming we don't already have something like this).